README.rst

```diff
@@ -232,3 +232,48 @@ customization - the ``zyte_api`` meta key can be set to ``True`` or ``{}``:
             #     'download_slot': 'quotes.toscrape.com'
             # }
 
+Customizing the retry policy
+----------------------------
+
+API requests are retried automatically using the default retry policy of
+`python-zyte-api`_.
```

Comment:
Optional: perhaps it might be convenient if we could add a link to https://github.com/zytedata/python-zyte-api/blob/main/zyte_api/aio/retry.py#L61-L131 so users could immediately know what to subclass and its existing behavior.